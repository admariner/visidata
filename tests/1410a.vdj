#!vd -p
{"sheet": "global", "col": "", "row": "numeric_binning", "longname": "set-option", "input": "True", "keystrokes": "", "comment": ""}
{"sheet": "", "col": "", "row": "", "longname": "open-file", "input": "sample_data/benchmark.csv", "keystrokes": "o", "comment": ""}
{"sheet": "benchmark", "col": "Item", "row": "12", "longname": "delete-cell", "input": "", "keystrokes": "zd", "comment": "delete current cell (set to None)"}
{"sheet": "benchmark", "col": "Item", "row": "20", "longname": "delete-cell", "input": "", "keystrokes": "zd", "comment": "delete current cell (set to None)"}
{"sheet": "benchmark", "col": "Item", "row": "23", "longname": "delete-cell", "input": "", "keystrokes": "zd", "comment": "delete current cell (set to None)"}
{"sheet": "benchmark", "col": "Item", "row": "27", "longname": "delete-cell", "input": "", "keystrokes": "zd", "comment": "delete current cell (set to None)"}
{"sheet": "benchmark", "col": "Item", "row": "", "longname": "freq-col", "input": "", "keystrokes": "Shift+F", "comment": "open Frequency Table grouped on current column, with aggregations of other columns"}
{"sheet": "benchmark_Item_freq", "col": "Item", "row": "", "longname": "sort-asc", "input": "", "keystrokes": "[", "comment": "sort ascending by current column; replace any existing sort criteria"}
