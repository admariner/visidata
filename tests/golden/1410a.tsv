Item	count	percent	histogram
	4	7.84	■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■
BFF Oh My Gravy! Beef & Chicken 2.8oz	1	1.96	■■■■■■■
BFF Oh My Gravy! Beef & Salmon 2.8oz	1	1.96	■■■■■■■
BFF Oh My Gravy! Chicken & Pumpkin 2.8oz	2	3.92	■■■■■■■■■■■■■■■
BFF Oh My Gravy! Chicken & Salmon 2.8oz	1	1.96	■■■■■■■
BFF Oh My Gravy! Chicken & Shrimp 2.8oz	1	1.96	■■■■■■■
BFF Oh My Gravy! Duck & Salmon 2.8oz	1	1.96	■■■■■■■
BFF Oh My Gravy! Duck & Tuna 2.8oz	1	1.96	■■■■■■■
BFF Oh My Gravy! Lamb & Tuna 2.8oz	1	1.96	■■■■■■■
Cat, Korat (Felis catus)	1	1.96	■■■■■■■
Cat, Maine Coon (Felix catus)	1	1.96	■■■■■■■
Cat, Scottish Fold (Felis catus)	1	1.96	■■■■■■■
Crickets, Adult Live (Gryllus assimilis)	1	1.96	■■■■■■■
Dog, Golden Retriever (Canis lupus familiaris)	2	3.92	■■■■■■■■■■■■■■■
Fire Extinguisher, kitchen-rated	1	1.96	■■■■■■■
Food, Adult Cat - 3.5 oz	2	3.92	■■■■■■■■■■■■■■■
Food, Adult Cat 3.5 oz	1	1.96	■■■■■■■
Food, Adult Dog - 5kg	1	1.96	■■■■■■■
Food, Kitten 3kg	2	3.92	■■■■■■■■■■■■■■■
Food, Pangolin	1	1.96	■■■■■■■
Food, Premium Wet Cat - 3.5 oz	1	1.96	■■■■■■■
Food, Quoll	1	1.96	■■■■■■■
Food, Rhinocerous - 50kg	1	1.96	■■■■■■■
Food, Salamander	1	1.96	■■■■■■■
Food, Senior Wet Cat - 3 oz	5	9.80	■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■■
Food, Shark - 10 kg	1	1.96	■■■■■■■
Food, Spider	1	1.96	■■■■■■■
Forti Diet Prohealth Mouse/Rat 3lbs	1	1.96	■■■■■■■
Goat, American Pygmy (Capra hircus)	1	1.96	■■■■■■■
Kingsnake, California (Lampropeltis getula)	1	1.96	■■■■■■■
Laser Pointer	1	1.96	■■■■■■■
Lizard, Spinytail (Uromastyx ornatus)	1	1.96	■■■■■■■
Mealworms, Large (Tenebrio molitor) 100ct	1	1.96	■■■■■■■
Monster, Rust (Monstrus gygaxus)	1	1.96	■■■■■■■
Mouse, Pinky (Mus musculus)	2	3.92	■■■■■■■■■■■■■■■
Parakeet, Blue (Melopsittacus undulatus)	1	1.96	■■■■■■■
Parrot, Norwegian Blue (Mopsitta tanta)	1	1.96	■■■■■■■
Rabbit (Oryctolagus cuniculus)	2	3.92	■■■■■■■■■■■■■■■
